agent_build/package_builder/linux/deb_or_rpm/install-scripts/postinstall.sh

```diff
@@ -159,7 +159,7 @@ ensure_path_not_readable_by_others() {
   ensure_path_other_permissions "${file_path}" "0"
 }
 
-check_python_version
+#check_python_version
```

Comment:
Uncomment ?

Reply:
The plan was to remove the "Python checking" things in pre-post scripts since we don't have to rely on the system Python anymore.